samples/local_attestation/README.md

```diff
@@ -203,24 +262,47 @@ See [here](https://github.com/openenclave/openenclave/tree/master/docs/MbedtlsSu
 
 ## Build and run
 
+### Linux
+
 Note that there are two different build systems supported, one using GNU Make and
 `pkg-config`, the other using CMake.
 
-### CMake
+#### CMake
 
 This uses the CMake package provided by the Open Enclave SDK.
 
 ```bash
-cd local_attestation
+cd attestation
 mkdir build && cd build
 cmake ..
 make run
 ```
 
-### GNU Make
+To use the SGX remote functionality, instead of the SGX local functionality,
+use `make runsgxremote` instead of `make run`.
```

Comment:
Can we make it so that `make run` runs both variants and add a `make runsgxlocal` for the local variant specifically?

Reply:
Done, look good now?

> Can we make it so that `make run` runs both variants and add a `make runsgxlocal` for the local variant specifically?

Done